ci/check-params-env.sh

```diff
@@ -0,0 +1,330 @@
+#!/bin/bash
```

Comment:
This is a great addition to the notebooks repo! 

I tested it locally, works as expected in both cases. :slightly_smiling_face: 

Matched
```
....
....
Validation of 'manifests/base/params.env' was successful! Congrats :)
```
 
Mismached
```
Checking format of: 'odh-minimal-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:f59aa0090a316878e51dbc89234eb3cee189839120a6ab619936872e966b1d5ae'
Checking metadata for image 'odh-minimal-notebook-image-n-1' with URL 'quay.io/opendatahub/workbench-images@sha256:f59aa0090a316878e51dbc89234eb3cee189839120a6ab619936872e966b1d5ae'
FATA[0000] Error parsing image name "docker://quay.io/opendatahub/workbench-images@sha256:f59aa0090a316878e51dbc89234eb3cee189839120a6ab619936872e966b1d5ae": invalid checksum digest length 
Couldn't download image metadata with skopeo tool!
ERROR: Image definition for 'odh-minimal-notebook-image-n-1' isn't okay!
....
....
```

/LGTM